docker/oso-saml-sso/src/start.sh

```diff
@@ -13,6 +13,7 @@ fi
 
 echo user:x:$(id -u):0:USER:/root:/bin/bash >> /etc/passwd
 echo group:x:$(id -G | awk '{print $2}'):user >> /etc/group
+reconf_touchfile="/configdata/reconfigure_successful"
```

Comment:
Probably better to just have the bash script pass in the file name and have the ansible playbook use the passed in one, if provided, and its configured default, if not.